polkit/pid_start_time.go

```diff
@@ -0,0 +1,64 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build linux
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package polkit
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// getStartTimeForPid determines the start time for a given process ID
+//
+// The implementation is intended to be compatible with polkit:
+//    https://cgit.freedesktop.org/polkit/tree/src/polkit/polkitunixprocess.c
+func getStartTimeForPid(pid uint32) (uint64, error) {
+	filename := fmt.Sprintf("/proc/%d/stat", pid)
+	file, err := os.Open(filename)
+	if err != nil {
+		return 0, err
+	}
```

Comment:
Is there a `defer file.Close()` missing here? Looking at the code it might be simple to just use:
```
data, err := ioutil.ReadFile(fmt.Sprintf("/proc/%d/stat", pid)
...
``` 
in lines 36-41 (which is slightly shorter and deals with the close for you).